modules/cms/classes/Page.php

```diff
@@ -62,25 +62,25 @@ public function __construct(array $attributes = [])
         ];
     }
 
-    protected function parseSettings()
+    protected function parseSettings(): void
     {
     }
 
     /**
      * Returns name of a PHP class to us a parent for the PHP class created for the object's PHP section.
      * @return mixed Returns the class name or null.
      */
-    public function getCodeClassParent()
+    public function getCodeClassParent(): ?string
     {
-        return '\Cms\Classes\PageCode';
+        return PageCode::class;
     }
 
     /**
      * Returns a list of layouts available in the theme.
      * This method is used by the form widget.
      * @return array Returns an array of strings.
      */
-    public function getLayoutOptions()
+    public function getLayoutOptions(): array
```

Comment:
The formatting we will go for declaring return types will be `methodName() : type`, not `methodName(): type`